lib/dal/BUCK

```diff
@@ -106,6 +106,7 @@ rust_test(
     ]),
     env = {
         "CARGO_PKG_NAME": "integration",
+        "RUSTC_BOOTSTRAP": "1",
```

Comment:
Bleh but, yeah, we need it.

Reply:
Agree, I couldn't quite figure out how to add in-line comments within the buck file so I improved the commentary on the commit to include why I added it, hopefully that helps someone in the future